### PR TITLE
Promote NMCs to approvers for active orgs

### PR DESCRIPTION
**Pull request by nikhita**
The current New Membership Coordinators have been _amazing_ at working through all the incoming membership requests. They've also been very careful around adding new members and have not been hesitant to ask questions when in doubt.

I think it's time we promote @ameukam @palnabarun and @savitharaghunathan to approvers! :tada: :tada: :tada: 

We also discussed this in the GitHub Management today and there were lots of +1s!

/hold
for ack by other approvers

/cc @kubernetes/owners 
/assign @cblecker @mrbobbytables @spiffxp

### CODE REVIEW

**Comment by dims** [comment]
yay!

**Review by spiffxp — APPROVED** [review]
/lgtm
/meowvie

**Comment by nikhita** [comment]
We have lgtms from two other approvers and we also had consensus in the meeting.

/hodl cancel

**Comment by nikhita** [comment]
> /hodl cancel

gah, pre-coffee brain!

/hold cancel

**Comment by palnabarun** [comment]
Woooooot! 😸 

Thank you, everyone! 😍 

/honk

**Pull request merged**
